projects/demo/src/modules/components/accordion/accordion.template.html

```diff
@@ -80,6 +81,19 @@
                         </div>
                     </ng-template>
                 </tui-accordion-item>
+                <tui-accordion-item
```

Comment:
Sorry, don't understand the necessity of this new css-variable.

User has direct access to `tui-accordion-item`. Could it set all required margins by yourself ?

Reply:
difficult 

https://stackblitz.com/edit/angular-a1bcwx?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.less

Reply:
Maybe we can make it not difficult, instead of expanding the API?

Reply:
sure